src/CMakeLists.txt

```diff
@@ -34,4 +34,8 @@ add_library(vbkutils
         $<TARGET_OBJECTS:sha256>
         $<TARGET_OBJECTS:bigdecimal>
         $<TARGET_OBJECTS:vblake>
-        )
\ No newline at end of file
+        )
+        
+if(WITH_ROCKSDB)
+    target_link_libraries(vbkutils rocksdb)
+endif()
```

Comment:
We decided not to link rocksdb against vbkutils. Link rocksdb separately, if possible.